docs/contributing/ast/python3_12.ast

```diff
@@ -0,0 +1,185 @@
+-- Python 3.12 AST
+-- ASDL's 4 builtin types are:
+-- identifier, int, string, constant
+
+module Python version "3.12"
```

Comment:
I think it is a bit early for this document. It may still change during the release process. I am still unsure why we keep copying these AST documents instead of linking to them.

Reply:
Well first the seceond question: 
> I am still unsure why we keep copying these AST documents instead of linking to them.

Well we also do link, but that does not give any real benefits. We do copy those files because then we do have the changes_from<old version>to<new version>.rst files that includes a diff for us in the documentation. This way we directly see any changes within the AST grammar. That is extremely helpful for a systematic and structered maintanance approach. 

> I think it is a bit early for this document. It may still change during the release process.
Yes it will change during the release process, but that does not make it wrong to copy it as early as possible. The currently copied is actually the version from 3.11 as a base, so thzat we could later also quickly see in our editors if there has been any changes on an update. 
This Document should be adapted while checking an upcoming release candidate of a new Python version.